workshop1/01-setup.md

```diff
@@ -4,60 +4,127 @@
+   mentors!
 
-1. Commit your changes by following [these instructions](../git-cheatsheet.md#commit).
+   In the terminal you started, run the following to create the application:
 
-2. With all the changes committed, the next step is to Push the code: check the steps [on the cheatsheet](../git-cheatsheet.md#push).
+       npx create-react-app . --typescript
+       npm start
 
-3. Now that the code is available at GitHub, let's create a [Pull Request](https://help.github.com/en/articles/about-pull-requests) by following the instructions [here](https://help.github.com/en/articles/creating-a-pull-request#creating-the-pull-request).
-  * Add a few people from the room as your reviewers: At this stage, we will only check if the files in the Pull Request you're reviewing are the same as your files. If all the steps were followed correctly, everyone should have the same generated code, therefore, there's no need for a deeper review.
-  * Once you get at least one approval, [merge](https://help.github.com/en/articles/merging-a-pull-request#merging-a-pull-request-on-github) the PullRequest. Select the `Merge pull request`, from the drop-down list (item 3 of the referred documentation).
+    If you're interested in what `create-react-app` is doing, you can [read
+    its "Getting Started"
+    guide](https://facebook.github.io/create-react-app/docs/getting-started).
+
+4. If it doesn't open automatically, open http://localhost:3000/ in your
+   browser to see the running application. (It doesn't do much yet!)
+
+### Commit the code changes and push to GitHub
+
+So far, we've generated a basic react app, which we will build on to create
+the animal shelter front-end. If you look at your repository folder now,
+you'll see a lot of files that were generated by `create-react-app`. This is
+what we'd call "scaffolding" or "boilerplate" code. It's the foundation on
+which we'll build our application.
+
+We'll commit and push this code to our master branch now for two reasons:
+
+1. This is a checkpoint for us: we have stable code which starts up a basic
+   React app that we can return to if needed.
+2. We will now start customizing this code to our needs, and when we review
+   our customizations later, we want to be able to clearly distinguish which
+   code we wrote and which of it was generated.
+
+To commit and push your changes:
+
+1. [Read the guidance on staging changes and committing in the Git cheatsheet](../git-cheatsheet.md#commit).
+
+   For example, you could open a new terminal using `Terminal → New Terminal` and run:
+
+       git add .
+       git commit
+
+2. [Read the guidance on pushing your changes in the Git cheatsheet](../git-cheatsheet.md#push).
+
+   For example, in a terminal, you could run:
+
+       git push
+
+**N.B.** Normally, we'd push to a branch rather than directly to the `master`
+branch when pushing code, so that we can easily submit our changes for review
```

Comment:
Small one: not sure if I got it wrong, but I think the last bit of the sentence can be removed?

Normally, we'd push to a branch rather than directly to the `master` branch ~when pushing code~

Reply:
Well spotted! Will fix that :)